proto/contimage/contimage.proto

```diff
@@ -49,6 +49,8 @@ message ContainerImage {
     }
     History         history   = 5;
   }
-  repeated ContainerImageLayer layers  = 10;
-  google.protobuf.Timestamp    builtAt = 11;
+  repeated ContainerImageLayer layers = 10;
+
+  google.protobuf.Timestamp          builtAt     = 11;
+  optional google.protobuf.Timestamp publishedAt = 13;
```

Comment:
Should this be 12?

Reply:
12 is already used

Reply:
yeah, it's already used by the `dd_tags` field. I've originally set it to 12 and got a compilation error 😄 
https://github.com/DataDog/agent-payload/blob/master/proto/contimage/contimage.proto#L20

Reply:
Ah, the 12 is hiding in the single-digit IDs def 😅